lib/ui/pointer.dart

```diff
@@ -34,6 +34,9 @@ enum PointerChange {
 
   /// The pointer has stopped making contact with the device.
   up,
+
+  /// The pointer has scrolled
+  scroll,
```

Comment:
Why did you change this back? This is incompatible with the second part of the change I'm working on, to support trackpad gesture scrolling.

My intent is to finish that patch and then put both of them up for review so that the reason for the design choices I made was clear. I think it would make a lot more sense to wait and do it that way than to try to land this one and make changes to it without being able to see how that affects the whole design.